src/trc.py

```diff
@@ -195,23 +200,45 @@ def _import_from_c3d(self, filename):
             # Set file header values.
             self['DataRate'] = reader.header.frame_rate
             self['CameraRate'] = reader.header.frame_rate
-            self['NumFrames'] = reader.header.last_frame
-            self['NumMarkers'] = reader.get('POINT').get('USED').int16_value
+            self['NumFrames'] = reader.header.last_frame - reader.header.first_frame + 1
             self['Units'] = reader.get('POINT').get('UNITS').string_value
             self['OrigDataRate'] = reader.header.frame_rate
             self['OrigDataStartFrame'] = reader.header.first_frame
             self['OrigNumFrames'] = reader.header.last_frame - reader.header.first_frame + 1
 
-            # Set data column labels.
-            self['Markers'] = [label.strip() for label in reader.point_labels]
-
             # Set frame numbers.
             self['Frame#'] = [i for i in range(reader.header.first_frame, reader.header.last_frame + 1)]
 
+            # Filter out model outputs (Angles, Forces, Moments, Powers, Scalars) from point labels.
+            point_group = reader.get('POINT')
+            model_outputs = set()
+            for param in ['ANGLES', 'FORCES', 'MOMENTS', 'POWERS', 'SCALARS']:
+                if param in point_group.param_keys():
+                    model_outputs.update(point_group.get(param).string_array)
+            point_labels = []
+            for param in ['LABELS', 'LABELS2']:
+                if param in point_group.param_keys():
+                    filtered_labels = [None if label in model_outputs else label.strip() for label in point_group.get(param).string_array]
+                    point_labels.extend(filtered_labels)
```

Comment:
This should be added as a keyword argument so users can override and change this list of filtered data.  Something like
```
filter_output=None
if filter_output is None:
  filter_output = ['ANGLES', 'FORCES', 'MOMENTS', 'POWERS', 'SCALARS']
```

same for `params`.

Reply:
Just added this.